src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml

```diff
@@ -31,6 +31,6 @@
             </HBox>
         </VBox>
     </content>
+    <ButtonType fx:id="applyButton" buttonData="OK_DONE" text="%Apply" />
```

Comment:
The order of buttons is OS depdenend. @tobiasdiez Does one have apply a special CSS class here? It does not seem to be applied correclty here (refs https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonType.html#APPLY)

Reply:
I have checked another dialog class `Customize EntryTypeDialog` and i wanted to keep the pattern, but right now i checked it again and seems to be unused.

Reply:
The order is displayed here: https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonBar.html

ButtonType = APPLY is indeed wrong, as this is for a button that saves the current state but leaves the dialog open. OK_DONE is correct. I would maybe change the display text to `%Save` to make it clearer and prevent confusion.

Reply:
Yes the id can be safely deleted. It's only needed if the code uses `lookupButton`